packages/storybook-utils/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@web/storybook-utils",
+  "version": "0.0.0",
```

Comment:
should we just release this as v1? we have a bunch of packages to indefinitely tend to stay on 0.0.0 😅

Reply:
I thought it was "the way" :D hahaha
but no problem, will make it 1.0, I don't have a problem going to 2.0 if needed later

Reply:
I changed the changeset to "major"

Reply:
> I thought it was "the way" :D hahaha

yeah historically we did tend to do this, but I think for a utils package like this its fine to just go 1.0.0